### DISCUSSION TITLE
config question..

### DISCUSSION THREAD

**Discussion by Tigger86** [blakeblackshear/frigate#14604] [General]
Hello,
I'm a bit up in the air...
I already have a Go2RTC instance running with a camera as a test.
I have integrated this with an RTSP link.

Now I would like to combine the whole thing with frigate.
in the end i don't need recording, tracking, or object detection, because i already have my synology surveilance running for this.
i only need frigate to duplicate a stream from an ipcam for my mobile devices.
but what is the smartest way to do this ? can anyone help me ?
i have set up frigate so far... except for a config.yml
please help.
Marcell

**Comment by NickM-27** [comment]
It doesn't sound like you should be using Frigate, just use go2rtc

**Reply by Tigger86** [reply]
but i have the problem that when i open a stream and it is running. the second stream at the latest no longer works

**Reply by NickM-27** [reply]
That's a config or device issue, it would be the exact same trying to use go2rtc in frigate except more resources would be used doing other things that you don't want
